content/en/docs/ops/configuration/traffic-management/multicluster/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Multi-cluster Traffic Management
+description: How to configure how traffic is distributed among clusters in the mesh.
+weight: 70
+keywords: [traffic-management,multicluster]
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Within a multicluster mesh, traffic rules specific to the cluster topology may be desirable. This document describes
+a few ways to manage traffic in a multicluster mesh. Before reading this guide:
+
+1. Read [Deployment Models](/docs/ops/deployment/deployment-models/#multiple-clusters)
+1. Make sure your deployed services follow the concept of {{< gloss "namespace sameness" >}}namespace sameness{{< /gloss >}}.
+
+## Keeping traffic in-cluster
+
+In some cases the default cross-cluster load balancing behavior is not desirable. To keep traffic "cluster-local" (i.e.
+traffic sent from `cluster-a` will only reach destinations in `cluster-a`), mark hostnames or wildcards as `clusterLocal`
+using [`MeshConfig.serviceSettings`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ServiceSettings-Settings).
+
+{{< tabset category-name="meshconfig" >}}
+
+{{< tab name="per-service" category-value="service" >}}
+
+{{< text yaml >}}
```

Comment:
Should give more context. What CRD is this, and were is it applied?

Reply:
Will do.